vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -369,6 +363,18 @@ void TouchEventHandler::UpdatePointersInViews(
 }
 
 winrt::Microsoft::ReactNative::JSValue TouchEventHandler::GetPointerJson(const ReactPointer &pointer, int64_t target) {
+  // defines button payoad, follows https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button
+  int button = -1;
+  if (pointer.isLeftButton) {
+    button = 0;
+  } else if (pointer.isMiddleButton) {
+    button = 1;
+  } else if (pointer.isRightButton || pointer.isBarrelButton) {
+    button = 2;
+  } else if (pointer.isEraser) {
+    button = 5;
```

Comment:
Recommend giving these things names and using the names here so readers don't have to look for it:
```cpp
// defines button payload, follows https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button
enum class MouseEventButtonKind {
  Main = 0,
  Auxiliary = 1,
  Secondary = 2,
  BrowserBack  = 3,
  BrowserForward = 4,
  Eraser = 5
};
```

then you can use the values like:
```cpp
  button = MouseEventButtonKind::Secondary
```

Reply:
Added! Didn't do BrowserBack or BrowserFoward since we don't define them in the payload.